src/tests/terminal/test_colors.f90

```diff
@@ -0,0 +1,106 @@
+! SPDX-Identifier: MIT
+
+module test_colors
+    use stdlib_colors
```

Comment:
I think we should be using explicit imports, that is, list all the things to import here.